### ISSUE TITLE
6.F. Preparing the Ballot (candidate information collection)

### ISSUE THREAD

**Issue by kgrons** [amiaopensource/amia-elections#20]
Propose to update language to **specify how the Committee must obtain the required information and documents IN THE SAME MANNER FROM ALL CANDIDATES**.

This may include:
- Consistent and documented information collection and delivery procedures
- Oversight in the collection and delivery process (maybe by staff in the Office?)

Specifying this will help reinforce the equal treatment of all candidates and candidate statements and provide an opportunity to explore more accountable procedures for candidate self-description.

**Comment by michelewinn** [comment]
Agree. There is already a degree of oversight from the committee, committee chair, and the office, but there should be language that ensures consistency.

**Comment by bpgraney** [comment]
As long as all candidates are presented equally with the opportunity to submit these statements for distribution through AMIA, is it necessary to make it a requirement to run for election?  Of course it's to the benefit of the candidates to make use of this opportunity, but if they decide against it—or if they fail to meet the submission deadline—I don't see why they should be disqualified from the ballot on that basis.

For reference, see Cali. Corp. Code [5523](http://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=CORP&sectionNum=5523.).
